Transmission.xcodeproj/project.pbxproj

```diff
@@ -1497,6 +1498,7 @@
 				0A6169A60FE5C9A200C66CE6 /* bitfield.h */,
 				1BB44E07B1B52E28291B4E30 /* file-piece-map.cc */,
 				1BB44E07B1B52E28291B4E31 /* file-piece-map.h */,
+				E71A5564279C2DD600EBFA1E /* tr-assert.mm */,
 				C1425B321EE9C5EA001DB85F /* tr-assert.cc */,
```

Comment:
I see tr-assert.cc is still referenced in a couple of places, are you sure it's not compiled?

Reply:
Yes it's in the project but not compiled.